src/components/content/CCDA/ccda_viewer/components/Section/Section.tsx

```diff
@@ -5,16 +5,21 @@ import { useState } from "react";
 import { fixHtml } from "../../helpers";
 import { SectionType } from "../../types";
 import "../../../styles.scss";
+import { useAnalytics } from "@/components/core/providers/analytics/use-analytics";
 
 export const Section = ({
   title,
   humanReadable,
 }: Omit<SectionType, "code">): JSX.Element | null => {
   const [isOpen, setIsOpen] = useState(true);
+  const { trackInteraction } = useAnalytics();
 
   if (!humanReadable) return null;
 
-  const handleClick = () => setIsOpen(!isOpen);
+  const handleClick = () => {
+    setIsOpen(!isOpen);
+    trackInteraction(isOpen ? "collapse_document_section" : "expand_document_section");
+  };
```

Comment:
We could make the event of clicking the document section a single event where "collapse" and "expand" are metadata on the event.
  
```suggestion
  const handleClick = () => {
    setIsOpen(!isOpen);
    trackInteraction("toggle_document_section", {
      direction: isOpen ? "collapse" : "expand",
    });
  };
```